pygmt/tests/test_grdimage.py

```diff
@@ -159,7 +159,7 @@ def test_grdimage_over_dateline(xrgrid):
     fig = Figure()
     assert xrgrid.gmt.registration == 0  # gridline registration
     xrgrid.gmt.gtype = 1  # geographic coordinate system
-    fig.grdimage(grid=xrgrid, region="g", projection="A0/0/1c", V="i")
+    fig.grdimage(grid=xrgrid, region="g", projection="A0/0/1c", verbose="i")
```

Comment:
The verbosity can be removed here I think. It was put in place because #375 was bugging us for a long time, but I think things have been stable since GMT 6.2.
```suggestion
    fig.grdimage(grid=xrgrid, region="g", projection="A0/0/1c")
```